src/main/jbake/content/developer/jms/common-jms-configuration.adoc

```diff
@@ -0,0 +1,95 @@
+= JMS Common Configuration
+:jbake-date: 2017-07-26
+:jbake-type: page
+:jbake-status: published
+:jbake-tomeepdf:
+
+See the link:advanced.html[JMS Advanced] for details on all configuration options for JMS.
+
+JBoss MQ
+[source,xml]
```

Comment:
See you can explain some of these settings. The JNDI parameters one is a key one for JBoss.